examples/llava/mtmd-cli.cpp

```diff
@@ -309,8 +317,15 @@ int main(int argc, char ** argv) {
             }
             g_is_generating = true;
             if (line.find("/image") == 0) {
```

Comment:
```suggestion
            if (line == "/image" || line.find("/image ") == 0) {
```
Just so we don't match `/imagefoobar`.